extensions/cpp/language-configuration.json

```diff
@@ -14,7 +14,8 @@
 		{ "open": "(", "close": ")" },
 		{ "open": "'", "close": "'", "notIn": ["string", "comment"] },
 		{ "open": "\"", "close": "\"", "notIn": ["string"] },
-		{ "open": "/*", "close": "*/", "notIn": ["string", "comment"] }
+		{ "open": "/*", "close": "*/", "notIn": ["string", "comment"] },
+		{ "open": "/**", "close": " */", "notIn": ["string"] }
```

Comment:
Should this also be `"notIn": ["string", "comment"]` and not just `"notIn": ["string"]` to match the `/*` rule?

Reply:
@alexr00 Yes, it needs to be `"notIn": ["string"]` , because when the user types the initial `/*` they're already in a comment, so it doesn't have any effect if `["string", "comment"]` is used -- this matches what is used by TypeScript for the exact same completion of `/**` to ` */`.

Reply:
Thanks for the explanation!